Generator.md

```diff
@@ -0,0 +1,339 @@
+# Installation
+
+## Add the NuGet Package to your project
+
+All you need to do, to install Agoda.Ioc.Generator is to add a NuGet reference pointing to the package Agoda.Ioc.Generator.
+
+## PackageReference
+
+``` xml
+<PackageReference Include="Agoda.Ioc.Generator" Version="1.1.128"   />
```

Comment:
Maybe we shouldn't mention explicit versions in the documentation?

Reply:
Fixed